snuba/migrations/migrate.py

```diff
@@ -45,7 +32,6 @@ def run(conn, dataset):
     schemas = []
     if dataset.get_table_writer():
         schemas.append(dataset.get_table_writer().get_schema())
-    schemas.append(dataset.get_dataset_schemas().get_read_schema())
```

Comment:
Why this removal ? If the datasets has multiple schemas out of which several read ones and one writer (like matviews that have 2 read schemas and a write schema) the read schemas would not be migrated.

Reply:
Yeah good point, I think this was an oversight.